src/Services/privateApiService.js

```diff
@@ -42,4 +42,27 @@ export const getRequest =  async (url, id = null) => {
     }
 }
 
-
+export const PutRequest = async(endpoint,id,body) => {
```

Comment:
El nombre de la función debería empezar con minúscula, y no le tengas miedo a los espacios jajaja

```suggestion
export const putRequest = async (endpoint, id, body) => {
```